src/tests/test_ggplot.py

```diff
@@ -431,7 +431,7 @@ def test_facet_wrap_stacked_histogram_cmap(diamonds_data):
 
 @_cleanup_cm()
 @image_comparison(
-    baseline_images=["test_facet_wrap_default_with_nulls"],
+    baseline_images=["facet_wrap_default_with_nulls"],
```

Comment:
does this mean the reference image is renamed? I see you deleted some images so please just confirm that you cleaned up any images that we don't need anymore

Reply:
yes I have only renamed this image file because all the files start with `facet..`. The other images have changed because now the order of the facets have changed to descending order

Reply:
I have cleaned up the image `test_facet_wrap_default_with_nulls.png`